core/domain/collection_domain.py

```diff
@@ -907,132 +912,71 @@ def exploration_ids(self):
 
     @property
     def init_exploration_ids(self):
-        """Returns a list of exploration IDs that are starting points for this
-        collection (ie, they require no prior skills to complete). The order
-        of these IDs is given by the order each respective exploration was
-        added to the collection.
+        """Returns the first element in the node list of the collection, which
+           corresponds to the first node that the user would encounter.
 
         Returns:
-            list(str). List of exploration IDs.
+            list(str). A single element list of the exploration ID of the
```

Comment:
Should this and callsites be updated to accept a str rather than a list? (Looking forward, I think that, although stories may be nonlinear, they would have a single starting point.)

Reply:
Done, changes all return types to string.